cypress/integration/test-helper.js

```diff
@@ -1,4 +1,4 @@
-const testResponseCode = (path, responseCode) => {
+const testNonHTMLResponseCode = (path, responseCode) => {
```

Comment:
If you're renaming this, something like `shouldHaveResponseCode` would be more in keeping with the other helper functions.

Reply:
It needs to explicitly say that this is for non HTML because it uses request rather than visit (and it was previously used incorrectly). Visit is needed for HTML and this is a core part of the refactor which rmvs the duplicate checks for 200 on HTML pages because the visit implicitly checks this already.